litellm/llms/clarifai.py

```diff
@@ -222,6 +221,50 @@ async def async_completion(
     )
     return model_response
 
+async def async_stream(
+    model: str,
+    prompt: str,
+    api_base: str,
+    custom_prompt_dict: dict,
+    model_response: ModelResponse,
+    print_verbose: Callable,
+    encoding,
+    api_key,
+    logging_obj,
+    data=None,
+    optional_params=None,
+    litellm_params=None,
+    logger_fn=None,
+    headers={},
+):
+
+    async_handler = AsyncHTTPHandler(timeout=httpx.Timeout(timeout=600.0, connect=5.0))
+    response = await async_handler.post(
+        url=model, headers=headers, data=json.dumps(data)
+    )
+
+    logging_obj.post_call(
+        input=prompt,
+        api_key=api_key,
+        original_response=response.text,
+        additional_args={"complete_input_dict": data},
+    )
+    ## RESPONSE OBJECT
+    try:
+        completion_response = response.iter_lines()
```

Comment:
this should be `aiter_lines` to return an async iterable

Reply:
My bad, Changed it to aiter and ran tests. It's running without any errors.

Reply:
@krrishdholakia 
Could you please take a loot at it ? 
TIA :)